style.css

```diff
@@ -341,4 +341,91 @@
     line-height: 20px;
     font-weight: 500;
   }
+
+  .contact {
+    width: 100%;
+    height: 779px;
+    background: #fff;
+    background-size: cover;
+  }
+
+  .contact-wrap {
+    position: relative;
+    height: 779px;
+    background-image: url(./images/contact_bg.png);
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 24px;
+  }
+
+  .contact-shapes-bg {
+    position: absolute;
+    top: 0;
+    right: 0;
+    height: 745px;
+  }
+
+  .contact-title {
+    font-size: 40px;
+    line-height: 52px;
+    font-weight: 700;
+    text-align: center;
+    color: #fff;
+  }
+
+  .contact-desc {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .contact-form ul {
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    margin-top: 20px;
+  }
+
+  .contact-form ul li {
+    list-style: none;
+  }
+
+  .contact-form ul li input {
+    border-radius: 8px;
+    margin-bottom: 20px;
+    width: 100%;
+    height: 48px;
+    border: 1px solid #cfd8dc;
+  }
+
+  .contact-form ul li input[type='submit'] {
+    width: 110px;
+    color: #6070ff;
+  }
+
+  .contact-form ul li input[type='submit']:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  .contact-form ul li input[type='submit']:visited {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .contact-form ul li input[type='submit']:disabled {
+    background: #5e6c84;
+    color: #5e6c84;
+  }
+
+  .contact-form ul li textarea {
+    border-radius: 8px;
+    margin-bottom: 20px;
+    width: 100%;
+    height: 170px;
+    border: 1px solid #cfd8dc;
+  }
```

Comment:
-Great job for sticking the given Figma design 👍 I think you missed a little black horizontal line at the bottom, so please add it to make your design more beautiful 👌
![2023-04-05_12-02](https://user-images.githubusercontent.com/52010789/230034258-75f4c8a3-28b8-478f-b416-6bb26dfe02cf.png)